src/components/footer/Footer.scss

```
@@ -1,3 +1,98 @@
-div {
-  font-size: 15px;
+@mixin base($justify) {
+  display: flex;
+  justify-content: $justify;
+}
+
+.footer {
+  position: relative;
+  margin-top: 100px;
+  transform: translateY(0%);
+
+  .footerNav {
+    @include base(space-around);
+    width: 100%;
+    padding-top: 15px;
+    padding-bottom: 15px;
+    border-top: 1px solid #c8c8c8;
+    border-bottom: 1px solid #eeeeee;
```

```suggestion
    border: solid #c8c8c8;
    border-width: 1px 0;
```

이렇게 한 번에 관리할 수도 있긴 한데, 이 정도는 취향차이일 수도 있겠네요.

---

확인했습니다 감사합니다!